index.html

```diff
@@ -1107,8 +1107,35 @@
 
 ## Principles for Notifications and Interruptions
 
-<aside class="issue">Connect intrusiveness to [=autonomy=], and write some principles.</aside>
+<aside class="issue">This is related to [=autonomy=].</aside>
 
+Notifications and other interruptive UI can be a powerful way to capture attention. 
+Depending on the operating system in use, a notification can appear outside of the
+browser context (for example, in a general notifications tray) or even cause a device to
+buzz or play an alert tone.
+Like all powerful features, notifications can be misused and can become an annoyance 
+or even used to manipulate behaviour and thus reduce [=autonomy=].
+
+<div class="practice">
+
+  <span class="practicelab" id="principle-notifications control">A [=user agent=] should help
+    users control notifications and other interruptive UI that can be used to manipulate behavior.</span>
+
+  [=User agents=] should provide UI that allows their users to audit which web [=sites=] have been granted 
+  permission to display alerts and to revoke these permissions.  
+  [=User agents=] should also apply some quality metric to the initial request for permissions to receive
+  notifications (for example, disallowing sites from requesting permission on first visit).
+  
```

Comment:
```suggestion
</div>
<div class="practice">
```

Otherwise Respec merges the two labels to the top of the block.